src/State.py

```diff
@@ -0,0 +1,53 @@
+
+from chess_enums import Space
+from chess_enums import Player
+from chess_enums import Piece
+import numpy
+
+
+class Action:
+    board: numpy.array  # of Piece, SpaceCount
+    king_space: numpy.array  # of Space, PlayerCount
+    king_moved: numpy.array  # of bool, PlayerCount
+    rookA_moved: numpy.array  # of bool, PlayerCount
+    rookH_moved: numpy.array  # of bool, PlayerCount
+    moves: list  # of Actions
+    player: Player
+    check: bool
+
+    def __init__(self) -> None:
+        board = {Space.___}
```

Comment:
Does this syntax work in Python? The way I found online to do it was "= [Piece.___]*len(Space)"

Reply:
Good point, I believe this was a copy pasta